config.py

```diff
@@ -10,6 +10,7 @@ class Config(object):
     SCHEMA = os.getenv("SCHEMA", "templatesvc")
     SQLALCHEMY_DATABASE_URI = os.getenv("SQLALCHEMY_DATABASE_URI",
```

Comment:
Can this be an SQLite3 URI for testing / travis?

Reply:
I use a Postgres docker instance to directly test its jsonb support.

Reply:
Looks like it doesn't translate well on travis - might need an alternative to run tests.

Reply:
It's an issue with the refactoring, creating my testing client isn't picking up the right config. Trying to sort it if not I will put up the commit without the refactoring.

Reply:
We could just spin up a docker postgres on travis?